manifests/n/NewzenMC/NewzenLauncher/1.0.2/NewzenMC.NewzenLauncher.locale.en-US.yaml

```diff
@@ -1,31 +1,28 @@
-# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
-
-PackageIdentifier: NewzenMC.NewzenLauncher
-PackageVersion: 1.0.2
-PackageLocale: en-US
-Publisher: TIEB62
-PublisherUrl: https://github.com/NewzenMC
-PublisherSupportUrl: https://github.com/NewzenMC/NewzenLauncher/issues
-# PrivacyUrl: 
-Author: NewzenMC
-PackageName: NewzenLauncher
-PackageUrl: https://github.com/NewzenMC/NewzenLauncher
-License: GPL3
-LicenseUrl: https://github.com/NewzenMC/NewzenLauncher/blob/main/LICENSE
-# Copyright: 
-# CopyrightUrl: 
-ShortDescription: Launcher Officiel de Newzen
-# Description: 
-# Moniker: 
-Tags:
-- electron
-- game
-- java
-- launcher
-- minecraft
-# Agreements: 
-# ReleaseNotes: 
-ReleaseNotesUrl: https://github.com/NewzenMC/NewzenLauncher/releases/tag/v1.0.2
-ManifestType: defaultLocale
-ManifestVersion: 1.1.0
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NewzenMC.NewzenLauncher
+PackageVersion: 1.0.2
+PackageLocale: en-US
+Publisher: TIEB62
+PublisherUrl: https://github.com/NewzenMC
+PublisherSupportUrl: https://github.com/NewzenMC/NewzenLauncher/issues
+Author: NewzenMC
+PackageName: NewzenLauncher
+PackageUrl: https://github.com/NewzenMC/NewzenLauncher
+License: GPL3
+LicenseUrl: https://github.com/NewzenMC/NewzenLauncher/blob/main/LICENSE
+ShortDescription: Launcher Officiel de Newzen
+Tags:
+- electron
+- game
+- java
+- launcher
+- minecraft
+ReleaseNotes: |-

```

Comment:
These should be translated to English for the en-US Locale

Reply:
Or change the locale to `fr-FR`.